components/Navbar.jsx

```diff
@@ -1,40 +1,88 @@
-import React from 'react'
-import { CiSearch } from "react-icons/ci";
-import { CiHeart } from "react-icons/ci";
+import React, { useState } from 'react'
+import { CiSearch,CiHeart } from "react-icons/ci";
 import { IoBagHandle } from "react-icons/io5";
+import { Menu } from 'lucide-react'
 import { CgProfile } from "react-icons/cg";
+import { Dancing_Script } from "next/font/google";
+import Sidemenu from './Sidemenu';
 import Link from 'next/link';
 import Image from 'next/image';
+const dancingScript = Dancing_Script({
```

Comment:
Better way would be to use the `link` tag and import it in the `_document.js` file under the pages folder.